src/main/kotlin/org/jitsi/nlj/rtcp/RtcpRrGenerator.kt

```diff
@@ -81,18 +81,20 @@ class RtcpRrGenerator(
             val streamStats = incomingStatisticsTracker.getSnapshot()
             val now = System.currentTimeMillis()
             val reportBlocks = mutableListOf<RtcpReportBlock>()
-            streamStats.ssrcStats.forEach { ssrc, statsSnapshot ->
+            streamStats.ssrcStats.forEach { (ssrc, statsSnapshot) ->
                 val senderInfo = senderInfos.computeIfAbsent(ssrc) {
                     SenderInfo()
                 }
-                val statsDelta = statsSnapshot.getDelta(senderInfo.statsSnapshot)
+                val fractionLost = statsSnapshot.computeFractionLost(senderInfo.statsSnapshot)
```

Comment:
why do we recompute `fractionLost` if it's in the snapshot already?

Reply:
fractionlost can't be computed from the snapshot because it's bound to an interval, not the cumulative stats